workshops/drumpad/README.md

```diff
@@ -0,0 +1,473 @@
+
+  // let is just another keyword that allows you to declare a variable in JavaScript.
+
+    let audio = new Audio(link);
+
+    // We load the sound from the link
+
+    audio.load();
+
+    // And we play it here
+
+    audio.play();
+
+}
+```
+
+To explain what we did above, we created a function called `play()`, and it receives a parameter, which is `link`. This is the link to the sound hosted on AWS (Amazon Web Services). Don't worry, you won't need to host your files by yourself. You can use mine. We then create an audio object and pass `link` to the object. Now we can just load the audio with the `load()` function and play it with the `play()` function.
+
+All we need to do now is add an on click event to each of our nested `divs` then we call the `play()` function and pass the link to the sound file as our parameter.
+
+Luckily the code required to playing our sounds is minimal. Let's navigate to our `index.html` file first, and then we will add the `onclick=""` HTML attribute, and inside we call the `play(link)` with the link inside like below:
+  
+```html
+
+<div  class="pad">
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/SD0025.mp3')">A</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/SD0010.mp3')">B</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/SD0000.mp3')">C</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/RS.mp3')">D</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/OH25.mp3')">E</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/MA.mp3')">F</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/CY0010.mp3')">G</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/CH.mp3')">H</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/CB.mp3')">I</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/BD0010.mp3')">J</div>
+
+  <div  class="box"  onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/BD0000.mp3')">K</div>
+
+  <div  class="box"  onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/CP.mp3')">L</div>
+
+</div>
+
+```
+
+Try it out by clicking the pads and see the magic!
+If everything works, your drum pad is complete!
+
+
+![Image](https://media.giphy.com/media/11sBLVxNs7v6WA/giphy.gif)
+ 
+## Publishing
+
+Now that you have finished building, you should share your beautiful creation with other people! Remember, it's as easy as giving them your URL! Don't forget to share it with me on Slack @emmanuel39hanks.
+
+## Resources
```

Comment:
instead of being put in this "resources" section at the end, all of these should be linked where appropriate earlier in the workshop as part of an explanation. i don't think the html, css, and javascript docs are directly relevant to the workshop, so they should be removed. the part where you introduce the onclick event in the workshop is a good place to link the onclick event docs. i also don't think you need to include javascript functions unless it's part of an explanation of what functions are within the workshop